src/site/static/eligibility.liquid

```diff
@@ -2033,8 +2033,44 @@ pageClass: "page-public-assistance"
   }
 
   function reducedFeeIdEligible() {
-    // TODO
-    return  false;
+    // https://www.dmv.ca.gov/portal/driver-licenses-identification-cards/identification-id-cards/
+    // https://www.icarol.info/ResourceView2.aspx?org=2225&agencynum=73919505
+    //
+    // DL 937 order form:
+    // https://www.dmv.ca.gov/portal/file/order-request-reduced-fee-or-no-fee-identification-card-program-dl-932-pdf/
+    //   Burton-Moscone-Bagley Citizens’ Income Security Act for Aged, Blind, and Disabled Californians (WIC §12000); is called WIC Section 12000 and does not have a seperate progam name, 
+    //   so could be part of the broader WIC benefits progam.
```

Comment:
The "WIC" in this case is the "welfare institutions code" and unassociated with the WIC program.  I believe the program described by WIC §12000 is SSP (the state portion of total SSI payments).